spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -381,6 +382,19 @@ public Session getSession() {
 		 */
 		private final Resource resource = new Resource();
 
+		/**
+		 * Configuring processor cache.
+		 */
+		private int processorCache = 200;
```

Comment:
We try to order things a bit more naturally. This property could be moved below `acceptCount`.